bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -35,45 +36,20 @@ class TalentPool(Cog, name="Talentpool"):
 
     def __init__(self, bot: Bot) -> None:
         self.bot = bot
-        self.reviewer = Reviewer(bot, self)
-        self.cache: Optional[defaultdict[dict]] = None
-        self.api_default_params = {'active': 'true', 'ordering': '-inserted_at'}
-
+        self.api = NominationAPI(bot.api_client)
+        self.reviewer = Reviewer(bot, self.api)
         # This lock lets us avoid cancelling the reviewer loop while the review code is running.
         self.autoreview_lock = asyncio.Lock()
 
     async def cog_load(self) -> None:
-        """Load user cache and maybe start autoreview loop."""
-        await self.refresh_cache()
```

Comment:
Question: I'm curious, why did we remove the cache ?

Reply:
It was refreshed pretty much every time it was accessed which made it pretty pointless, as if you make an API call every time anyway it defeats the point of a cache. There were also a bunch of times where it was refreshed even though it didn't need to be, so i'd expect this PR actually makes *less* API calls now.

It would be possible to reimplement it properly (in `_api.py` so it's more contained), although it's not a very high-throughput cog so it's not really necessary and is just extra code that needs to be maintaned and could have bugs.

The reason it was there in the first place is because it used to be a watchchannel (https://github.com/python-discord/bot/blob/main/bot/exts/moderation/watchchannels/_watchchannel.py) which tldr meant it needed to access api info for every message sent on the server so a cache was necessary.

Reply:
And do we have metrics of number of api calls / duplicated calls ?

Reply:
We have stats for requests the site gets by status code over time, but not where they're coming from afaik.